app/src/main/java/org/simple/clinic/drugs/selection/dosage/DosagePickerSheet.kt

```diff
@@ -28,7 +28,7 @@ import java.util.Locale
 import java.util.UUID
 import javax.inject.Inject
 
-class DosagePickerSheet : BottomSheetActivity() {
+class DosagePickerSheet : BottomSheetActivity(), DosagePickerUi {
```

Comment:
We are implementing the `DosagePickerUi` in `DosagePickerSheet` right? why does commit message mention extract? 😅

Reply:
Because the interface was extracted from an already existing screen.

Reply:
Ah, okay. 👍